### PR TITLE
Enhance footer links

### PR DESCRIPTION
**Pull request by djangocreation**
In response to issue #71

### CODE REVIEW

File: templates/_footer.haml [hunk 1]
```diff
@@ -8,7 +8,19 @@
...
   %a{href: "https://github.com/political-memory/political_memory/issues"}
     - trans "Bug tracking system"
+  &nbsp;
   %a{href: "https://github.com/political-memory/political_memory/issues/new"}
     - trans "Report a bug"
   %br
   {% trans "Provided by" %} <a href="http://www.laquadrature.net/">La Quadrature du Net</a>
+  %br
+  %hr
+  %a{href: "http://parltrack.euwiki.org/"}
```
Comment (njoyard) [inline templates/_footer.haml]: Maybe add something like "EP data provided by <a>Parltrack</a>" ?

Comment (njoyard) [inline templates/_footer.haml]: Maybe add something like "EP data provided by <a>Parltrack</a>" ?

File: templates/_footer.haml [hunk 2]
```diff
@@ -8,7 +8,19 @@
     <a href="http://www.gnu.org/licenses/agpl.html">GNU aGPLV3+</a>
   %a{href: "https://github.com/political-memory/political_memory/issues"}
     - trans "Bug tracking system"
```
Comment (njoyard) [inline templates/_footer.haml]: Also get rid of this link, "Report a bug" is enough

File: templates/_footer.haml [hunk 3]
```diff
@@ -8,7 +8,19 @@
...
+  &nbsp;
   %a{href: "https://github.com/political-memory/political_memory/issues/new"}
     - trans "Report a bug"
   %br
   {% trans "Provided by" %} <a href="http://www.laquadrature.net/">La Quadrature du Net</a>
+  %br
+  %hr
```
Comment (njoyard) [inline templates/_footer.haml]: I'm not that fond of the look with a HR. Maybe get rid of the BR and the HR and put your new links on the same link as the LQDN link ?
Also take care some links seem to include spaces when rendered (ie. an additional underlined space at the end of the link).

File: .gitignore [hunk 4]
```diff
@@ -18,3 +18,4 @@ pip-delete-this-directory.txt
 .dpl
 db.sqlite
 log/
+wsgi/static/
```
Comment (njoyard) [inline .gitignore]: That's a good idea but should not be in this PR :)
(We should do a cleanup pass on the repo I think)

File: memopol/tests/response_fixtures/RepresentativeListTest.test_page1_paginateby12_displaylist/content [hunk 5]
```diff
@@ -779,12 +779,27 @@
   <a href='https://github.com/political-memory/political_memory/issues'>
     Bug tracking system
   </a>
+  &nbsp;
   <a href='https://github.com/political-memory/political_memory/issues/new'>
     Report a bug
   </a>
   <br />
   Provided by <a href="http://www.laquadrature.net/">La Quadrature du Net</a>
+  <br />
```
Comment (jpic) [inline memopol/tests/response_fixtures/RepresentativeListTest.test_page1_paginateby12_displaylist/content]: Should we really be using brs here ?

File: memopol/tests/response_fixtures/RepresentativeListTest.test_page1_paginateby12_displaylist_searchjoly/content [hunk 6]
```diff
@@ -293,12 +293,27 @@
...
     Report a bug
   </a>
   <br />
   Provided by <a href="http://www.laquadrature.net/">La Quadrature du Net</a>
+  <br />
+  <hr />
+  <a href='http://parltrack.euwiki.org/'>
+    Parltrack
+  </a>
+  &nbsp;
```
Comment (jpic) [inline memopol/tests/response_fixtures/RepresentativeListTest.test_page1_paginateby12_displaylist_searchjoly/content]: Shouldn't we be using a CSS rule instead ?

**Comment by jpic** [comment]
I like @njoyard's comment:

> Maybe add something like "EP data provided by Parltrack" ?

Apart from that, I can live with how the rest of the PR is implemented. It's not perfect of course because it's built uppon legacy code and comments about usage of &nbsp; and brs and hrs are valid, but I don't think we should invest time into fixing that.

Instead, I propose to deploy this and invest time #57: V3 design implementation. @njoyard would that work for you ?

**Comment by jpic** [comment]
We need to be able to choose which blocks we want in django-responsediff to prevent duplicate tests, I'm implementing this.

**Comment by jpic** [comment]
This patch is better than what we have IMHO, I'd like to deploy it.

**Comment by njoyard** [comment]
Agreed ; the hr+br combination is an eye sore though ;) but i'll merge it and make a quick fix tonight.

**Pull request merged**